app/middlewares/validator.js

```diff
@@ -0,0 +1,39 @@
+const { check, validationResult } = require('express-validator/check'),
+  errors = require('../errors');
+
+const validateErrors = (req, res, next) => {
+  const validationErrors = validationResult(req);
+  if (!validationResult(req).isEmpty()) {
+    throw errors.validationFailed(validationErrors.array());
```

Comment:
I have the doubt if this throw works, could you please check.

Reply:
if it works it returns it this way
```suggestion
     "message": [
        {
            "value": "jonny.berrio@gmail.com",
            "msg": "The email does not belong to a Wolox domain.",
            "param": "email",
            "location": "body"
        }
    ],
    "internal_code": "validation_error"
}
```